test/server/server_test.cc

```diff
@@ -1024,33 +1024,6 @@ TEST_P(ServerInstanceImplTest, BootstrapNodeWithOptionsOverride) {
   EXPECT_EQ("bootstrap_sub_zone", server_->localInfo().node().locality().sub_zone());
 }
 
-// Validate server runtime is parsed from bootstrap and that we can read from
-// service cluster specified disk-based overrides.
-TEST_P(ServerInstanceImplTest, BootstrapRuntime) {
```

Comment:
Emm...this test doesn't seem a specific test for `server_->enableReusePortDefault()`, so I don't know if we should delete it directly.